R/simulateConsumerResource.R

```diff
@@ -175,10 +155,10 @@ simulateConsumerResource <- function(n_species, n_resources,
     outflow_rate = 0,
     volume = 1000,
     ...) {
-    t_dyn <- simulationTimes(t_end = t_end, ...)
+    t_dyn <- .simulationTimes(t_end = t_end, ...)
```

Comment:
just make sure that simulationTimes is not an exported function that is needed elsewhere.